packages/arcana-wallet/src/ArcanaWalletConnector.ts

```diff
@@ -0,0 +1,19 @@
+import { FuelWalletConnector } from '@fuel-connectors/fuel-wallet';
+import type { ConnectorMetadata } from 'fuels';
+import { APP_IMAGE } from './constants';
+
+export class ArcanaWalletConnector extends FuelWalletConnector {
```

Comment:
You can extend from `FuelConnector` since `FuelWalletConnector` is our official wallet. Keeping it as is will trigger our extension for everything.

```suggestion
export class ArcanaWalletConnector extends FuelConnector {
```

Reply:
Ignore this comment, I've just seen you guys used same communication protocol. Great work.